### DISCUSSION TITLE
Error aerender version

### DISCUSSION THREAD

**Discussion by Carrabba** [inlife/nexrender#861] [Q&A] [answered]
I tried to render my project but I get this error
How can I fix it
![Capture](https://user-images.githubusercontent.com/104204039/221827572-52f06546-8ce5-451a-8a1f-01ebf9a55e86.PNG)

**Comment by JamesBotterill** [comment, marked as answer]
It looks like you're looking to install script to Adobe After Effects CS6 when I believe the minimum supported version is Adobe After Effects 2015
